src/Injector.php

```diff
@@ -28,18 +28,12 @@ class Injector implements InjectorInterface
 {
     /** @var DefinitionInterface */
     protected $definition;
-
-    /** @var ContainerInterface */
-    protected $container;
-
-    /** @var DependencyResolverInterface */
-    protected $resolver;
-
-    /** @var ConfigInterface */
-    protected $config;
+    protected ContainerInterface $container;
+    protected DependencyResolverInterface $resolver;
+    protected ConfigInterface $config;
 
     /** @var string[] */
-    protected $instantiationStack = [];
+    protected array $instantiationStack = [];
```

Comment:
this is BC break, only `private` property that allowed the type to be changed when class is not final

Reply:
I see, does that mean the other protected properties I modified should be reverted back to use the docblocks?